styles.css

```diff
@@ -0,0 +1,96 @@
+/* Adding poppins fonts */
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  font-family: Poppins, sans-serif;
+}
+
+/* adding the main container */
+
+.container {
+  min-width: 375px;
+  max-width: 100%;
+  padding-bottom: 70px;
+  background-color: #fff;
+  background-repeat: no-repeat;
+  border-bottom-left-radius: 80px;
+  position: relative;
+  background-image: url(images/header-shapes-mobile.jpg);
+  background-size: cover;
+  margin: 24px;
+}
```

Comment:
- [ ] As requested by the previous reviewers kindly consider sticking to the Figma design as much as possible by following the margins (top, bottom, left, right) defined in the design. because If it's pretty and works, it will be a wonderful website.  check the images below and see all  missing in your design and try to fix that before requesting another review
- your design 👇 
<img width="425" alt="Screen Shot 2023-06-07 at 09 56 50" src="https://github.com/ausiemass/portfolio-austin/assets/60146030/99c224d5-6656-4ca7-b70a-2004426c2b91">

- Figma design 👇 
<img width="378" alt="Screen Shot 2023-06-07 at 09 59 43" src="https://github.com/ausiemass/portfolio-austin/assets/60146030/6090888c-dfec-426f-9c38-c0478b36ddf7">